server/controllers/usersController.js

```diff
@@ -1,9 +1,21 @@
-// CONTROLLER FOR USER MODEL
+const bcrypt = require("bcrypt");
 
-const db = require("../models/user");
+const connection = require("../dbConnection");
+const userSchema = require("../models/user");
 
-// Defining methods for the usersController
-module.exports = {
-  // define methods here
+const User = connection.model("User", userSchema);
 
+const SALT_ROUNDS = Number(process.env.SALT_ROUNDS);
```

Comment:
We want to make sure everyone is using consistent values for their env, so we'll need to sync about this and all the other env values in this PR at some point

Reply:
Yes, we discussed the same thing and came agreement to use slack to sync our `.env` files